google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java

```diff
@@ -3208,4 +3210,72 @@ public void testBucketLogging() throws ExecutionException, InterruptedException
       RemoteStorageHelper.forceDelete(storage, loggingBucket, 5, TimeUnit.SECONDS);
     }
   }
+
+  @Test
+  public void testSignedPostPolicyV4() {
```

Comment:
This is more of a unit test so I would say it's good to keep. I'd like to see a request similar to what you added into your documentation above:

```
PostFieldsV4 fields = PostFieldsV4.newBuilder().setAcl("public-read").build();
PostConditionsV4 conditions = PostConditionsV4.newBuilder().addContentTypeCondition(ConditionV4Type.MATCHES, "image/jpeg").build();

PostPolicyV4 policy = storage.generateSignedPostPolicyV4(
 BlobInfo.newBuilder("my-bucket", "my-object").build(),
 7, TimeUnit.DAYS, fields, conditions);

HttpClient client = HttpClientBuilder.create().build();
HttpPost request = new HttpPost(policy.getUrl());
for(Map.Entry<String, String> entry : policy.getFields().entrySet()) {
     request.addHeader(entry.getKey(), entry.getValue());
}
// Missing data to upload in request
client.execute(request);
```